packages/docsearch-react/src/AlgoliaLogo.tsx

```diff
@@ -18,7 +18,7 @@ export function AlgoliaLogo({ translations = {} }: AlgoliaLogoProps) {
       rel="noopener noreferrer"
     >
       <span className="DocSearch-Label">{searchByText}</span>
-      <svg width="77" height="19">
+      <svg width="77" height="19" focusable="false" aria-label="Algolia">
```

Comment:
I've done many tests with the screen reader and when the link is focused it says `Search by Algolia` so I believe it's good

Reply:
ah because it reads the link and the text inside it altogether I guess.
So if the person puts ${searchByText="Use elasticsearch not"} it will read `Use elasticsearch not Algolia` ? ahah